### ISSUE TITLE
French mortar calculations are removed

### ISSUE THREAD

**Issue by onmymomma001** [sh4rkman/SquadCalc#85]
after the recent update to the website i was wondering what happened to the French mortar mod calculations if they were affected by the squad update or not

**Comment by sh4rkman** [comment]
Yeah i was thinking that no servers was running the french mod atm, and i desactivated it, but i could be wrong.

What server(s) are currently running the mod ? If there is enough servers/online players i might reconsider
